_source/no-search/exceptions-deployments.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: article
+title: Deployment markers
+permalink: /user-guide/insights/exceptions/deployments.html
+flags:
+  logzio-plan: community
+  beta: true
+tags:
+  - insights
+  - exceptions
+  - deployment markers
+  - markers
+contributors:
+  - shalper
+---
+
+{% include page-info/early-access.md type="Beta" %}
+
+
+![Exceptions count in Kibana Discover](https://dytvr9ot2sszz.cloudfront.net/logz-docs/kibana-discover/deployments.png)
+
+#### Adding Deployment markers to Exception graphs
+
+You can send deployment logs by API to automatically correlate exceptions with service deployments directly in your Logz.io Exception graphs.
```

Comment:
```suggestion
You can send deployment logs by API to automatically correlate exceptions with service deployments directly in your Logz.io Exception tab.
```
maybe tab instead of graphs?